tests/fireflyClient.test.ts

```diff
@@ -393,6 +393,36 @@ describe("FireflyClient", () => {
       expect(data.response.data.length).to.be.gte(0);
     });
 
+    it("should get open orders of specific hashes", async () => {
```

Comment:
The 2 tests have the same name, I'd recommend changing this one. Maybe something like `"should handle get open orders of non-existent hashes"`